packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.scss

```diff
@@ -0,0 +1,108 @@
+@import '../../../../../../styles/shared.scss';
+
+.SelectList {
+  @include inputStyles;
+  font-size: 14px;
+  line-height: 14px !important;
+  border: 1px solid $neutral-200 !important;
+  border-radius: 6px !important;
+  min-height: 40px;
+  max-height: 40px;
+  padding: 10px 16px;
+
+  [class$='-control'] {
+    gap: 8px;
+    letter-spacing: 0.02em;
+    width: 100%;
+    flex-wrap: wrap;
+    align-items: flex-start !important;
+    cursor: text !important;
+
+    [class$='-ValueContainer'] {
+      padding: 0 !important;
+      max-height: 24px;
+      line-height: 14px;
+
+      [class$='-multiValue'] {
+        padding: 2px 8px;
+        border-radius: 6px;
+        margin: 0px !important;
+        margin-right: 8px !important;
+        background-color: $neutral-100;
+        // max-height: 24px;
```

Comment:
is this necessary? if not, please remove

Reply:
nice spot, will remove shortly

Reply:
[Done](https://github.com/hicommonwealth/commonwealth/pull/5086/commits/1ad60ace7ac529b5378418402efe2995001f0f2b)